pkg/summary/summary.go

```diff
@@ -287,15 +286,34 @@ func (s *Summarizer) updateWorkloads() error {
 	// map goldilocks-workload.name -> &controllerUtils.Workload{} for easy vpa lookup.
 	s.workloadForVPANamed = map[string]*controllerUtils.Workload{}
 	for _, w := range workloads {
-		w := w
-		controllerName := w.TopController.GetName()
-		vpaName := fmt.Sprintf("goldilocks-%s", controllerName)
-		s.workloadForVPANamed[vpaName] = &w
+		for _, v := range s.vpas {
+			w := w
+			v := v
```

Comment:
Honestly, I don't know. I was copying from the `w` :-D

Reply:
I don't think it is. I am guessing it was done for the workload because we end up using that workload, but for the vpa, we only need the name from the object, so it's probably not needed

Reply:
```suggestion
```